services/transcode/multitrack.go

```diff
@@ -35,11 +35,21 @@ func MultitrackMux(files paths.Files, outputPath paths.Path, cb ffmpeg.ProgressC
 
 	outputPath = outputPath.Append(files[0].Base() + ".mxf")
 
+	params = append(params, "-map", "v")
+	for i, _ := range files {
+		//params = append(params, "-map", fmt.Sprintf("%d", i))
+		t := i + 1
+		params = append(params, "-filter_complex", fmt.Sprintf("[%d:a:0]channelsplit=channel_layout=stereo[l%d][r%d]", t, t, t))
```

Comment:
are you sure ffmpeg supports passing more than 1 "-filter_complex"?

Reply:
Yes, this code was used to produce all 5 tracks for MF on saturday, but I ran it locally.